### DISCUSSION TITLE
allow other values than numbers

### DISCUSSION THREAD

**Discussion by Vince56190** [agronick/aa-torque#120] [Ideas]
I use more and more icons instead of numbers for my gauges (FAP regeneration indication, Gear engaged, Temperature alert etc.) but on the main dials, you can only put numbers. would it be possible to authorize the same typographies as the secondary gauges?For example, I can't put what I circled in red in the circular gauges.
![Capture d'écran 2024-02-25 082124](https://github.com/agronick/aa-torque/assets/147246549/e39113c3-e284-4941-ae3a-c871261658b7)
t I circled in red in the circular gauges.

**Comment by haydoyle** [comment]
How were you able to display the gear number?

**Comment by Vince56190** [comment]
Cela depends de votre voiture , moi c'est une Chrysler 300. il vous faut trouver le PID de votre voiture

**Comment by agronick** [comment]
The gauge library isn't really built to support that and I'm not trying to refactor it for that.
